dockstore-common/src/main/java/io/dockstore/common/PipHelper.java

```diff
@@ -22,7 +22,7 @@ private PipHelper() { }
      * @return              The most recently changed pip requirements file to the Dockstore client version (can be older, but not newer)
      */
     public static String convertSemVerToAvailableVersion(String semVerString) {
-        if (semVerString == null) {
+        if (semVerString == null || "development-build".equals(semVerString)) {
```

Comment:
Most likely overkill, but you could make public static with this value in dockstore-common, and then you would use that variable here and in MetadataApiServiceImpl.java.

Less work would be to have a comment here saying to match the value in MetadataApiServiceImpl.java, and vice-versa.

Or just leave it like this; it's only one string and it's not that hard to figure out.

Reply:
i originally considered using a constant but decided against making another file in commons and couldn't find a suitable file put the constant in.